src/main/kotlin/io/github/mojira/arisa/ModuleRegistry.kt

```diff
@@ -229,8 +230,8 @@ class ModuleRegistry(jiraClient: JiraClient, private val config: Config, private
         register(Modules.TransferVersions, TransferVersionsModule()) { issue ->
             AbstractTransferFieldModule.Request(
                 issue.key,
-                issue.getLinks(jiraClient, ::addAffectedVersionById, ::getVersionsGetField),
-                issue.versions.map { it.id }
+                issue.getLinks(jiraClient, ::addAffectedVersionById, ::getVersionsGetField.partially2 { Unit.right() }),
+                issue.getVersions { Unit.right() }
```

Comment:
What's with the hardcoded `Unit.right()`?
Is that unfinished code? It's supposed to return a mapped Version, IIRC

Reply:
It's passes as the `execute` function in `domain.Version` which is used to either add or delete the version from issue if I understand correctly. So I hardcoded it as `Unit.right()` here since we don't need either of that functions in this module.

Reply:
Right, we probably don't even *need* the remove by ID now. I'm cleaning this up a little

Reply:
Okay I understand this now, I am gonna work on #209 now and clean it up there

Reply:
Can archived versions be removed? I've heard about that they can only be added but not removed, but I'm not really sure if that's true or not. If we _can_ remove them, #157 might need the remove by ID function.